shaders/lang/ru_RU.lang

```diff
@@ -1,46 +1,105 @@
-option.SHADER_VERSION_LABEL = Bliss In-Dev Commit Number
-
-profile.QUALITY = Качество
-profile.PERFORMANCE = Представление
-
-screen.Sun_and_Moon_Colors = Цвет солнца/луны
-    option.sunColorR = Солнечный свет R
-	option.sunColorG = Солнечный свет G
-	option.sunColorB = Солнечный свет B
-	option.moonColorR = Лунный свет R
-	option.moonColorG = Лунный свет G
-	option.moonColorB = Лунный свет B
-	option.colortype = Цветовой тип
-	option.Sun_temp = Температура солнца
-	option.Moon_temp = Температура луны
-	
-	option.colortype.comment = 1 = RGB для цвета солнца/луны; 2 = черное тело
-
-screen.Misc_Settings = Разные настройки
-option.sunPathRotation = Угол наклона солнца/луны
-option.sun_illuminance = Свет Солнца
-option.moon_illuminance = Свет Луны
-option.MOONPHASE_BASED_MOONLIGHT = Уровень яркости фаз луны
-option.TRANSLUCENT_COLORED_SHADOWS = Полупрозрачные цветные тени
-option.OLD_LIGHTLEAK_FIX = Исправление утечки света
-
-option.RESOURCEPACK_SKY = Небо текстурпака
-    value.RESOURCEPACK_SKY.0 = Выкл.
-    value.RESOURCEPACK_SKY.1 = Небо + Шейдер солнца/луны
-    value.RESOURCEPACK_SKY.2 = Все небо
-
-screen.Waving_Stuff = Колыхание растительности
-    option.WAVY_PLANTS = Колыхание растительности
-    option.WAVY_STRENGTH = Интенсивность колыхания
-    option.WAVY_SPEED = Скорость колыхания
-
-screen.Direct_Light = Прямой свет
+ ____________________
+|                    |
+| By ShrikerChestTree|
+|   Version: 1.1.0   |
+|____________________|
+
+{~=~=~BLISS SHADERS~=~=~}
+
+option.SHADER_VERSION_LABEL = Bliss В-Разраб(In-Dev) Число Коммита
+    prefix.SHADER_VERSION_LABEL = КОММИТ #
+
+# если вы здесь, чтобы переводить, это шутка, они ничего не выполняют того, что говорят.
+# Okay)
+profile.QUALITY=Качество
+profile.PERFORMANCE=Производительность
+
+{#####_Misc. Settings_#####}
+
+screen.Misc_Settings = Настройка §cОтладочный§r/Экспериментальный
+
+|/////Sun and Moon Colors/////|
+
+screen.Sun_and_Moon_Colors = Цвет Солнца/Луны
+
+[-----sunPathRotation-----]
+
+option.sunPathRotation = Наклон Солнца
+
+[-----sun illuminance-----]
+
+option.sun_illuminance = Яркость Солнца
+
+[-----moon illuminance-----]
+
+option.moon_illuminance = Яркость Луны
+
+[-----Moonphase Based Moonlight-----]
+
+option.MOONPHASE_BASED_MOONLIGHT = Уровни яркости лунных фаз
+
+{#####_Settings For Supported Mods_#####}
+
+screen.Mod_support = Настройка Для Поддерживаемых Модов
+
+[-----ResourcePack Sky-----]
+
+option.RESOURCEPACK_SKY = Небо РесурсПака
```

Comment:
```suggestion
option.RESOURCEPACK_SKY = Небо из Ресурс Пака
```

Reply:
Where did you find the word “of” in “ResourcePack Sky” and why did you decide to make it separate?
I usually make the translation accaurate and double-check the original with the translation.
It's even labeled original up there and you still decided to make such a translation.

Reply:
At some points the original name is not shown and the ID name is shown instead